### PR TITLE
Add missing samples to ts/js recipe

### LINKED ISSUE #117: Verify Code Review Recipe for JavaScript and TypeScript
Guidance on how to CR the language and what tools can be used in the process to support the CR process

### PR DESCRIPTION
**Pull request by debugerr**
Closes #117 
- completed
1. build validation sampel
2. pre-commit hook sample

### CODE REVIEW

File: code-reviews/recipes/javascript-and-typescript.md [hunk 1]
```diff
@@ -118,15 +118,28 @@ By default, we use the following overrides should be added to the VS Code config
 
 ## Build Validation
 
-To automate this process in Azure Devops you can add the following snippet to you `azure-pipelines.yaml` file. This will lint any scripts in the `./scripts/` folder.
-
-> TODO: Add example of yaml for automating linting and testing
+To automate this process in Azure Devops you can add the following snippet to you pipeline definition yaml file. This will lint any scripts in the `./scripts/` folder.
```
Comment (tompaana) [inline code-reviews/recipes/javascript-and-typescript.md]: Typo: you -> your 

But how about: "As an example of automating linting in build validation process, the following snippet of an Azure DevOps pipeline definition YAML file will run a linter to inspect scripts in `./scripts` folder:"

File: code-reviews/recipes/javascript-and-typescript.md [hunk 2]
```diff
@@ -118,15 +118,28 @@ By default, we use the following overrides should be added to the VS Code config
...
+```
 
 ## Pre-commit hooks
 
 All developers should run `eslint` in a pre-commit hook to ensure standard formatting. We highly recommend using an editor integration like [vscode-eslint](https://github.com/Microsoft/vscode-eslint) to provide realtime feedback.
 
-> TODO: Show example
+1. Under `.git/hooks` rename `pre-commit.sample` to `pre-commit`
+1. Remove the existing sample code in that file
+1. There are various samples of scripts on gist that can be used. For example [pre-commit-eslint](https://gist.github.com/linhmtran168/2286aeafe747e78f53bf)
```
Comment (tompaana) [inline code-reviews/recipes/javascript-and-typescript.md]: Nit: Maybe abbreviate: "...used. For example" -> "...used, e.g."

File: code-reviews/recipes/javascript-and-typescript.md [hunk 3]
```diff
@@ -118,15 +118,28 @@ By default, we use the following overrides should be added to the VS Code config
...
+1. Adjust accordingly to include TypeScript files
```
Comment (tompaana) [inline code-reviews/recipes/javascript-and-typescript.md]: Nit: Adjust -> modify? Might be useful to elaborate this sentence a bit.

File: code-reviews/recipes/javascript-and-typescript.md [hunk 4]
```diff
@@ -118,15 +118,28 @@ By default, we use the following overrides should be added to the VS Code config
...
+1. `chmod +x .git/hooks/pre-commit`
```
Comment (tompaana) [inline code-reviews/recipes/javascript-and-typescript.md]: Nit: Preface with explanation, e.g. "Make the file executable: "

File: code-reviews/recipes/javascript-and-typescript.md (hunk 2 above)
Comment (TessFerrandez) [reply code-reviews/recipes/javascript-and-typescript.md]: or "There are many examples of scripts for this on gist, like [pre-commit-eslint](https://gist.github.com/linhmtran168/2286aeafe747e78f53bf)

**Pull request merged**